.github/workflows/main.yml

```diff
@@ -8,11 +8,9 @@ jobs:
   conda-mkdocs:
     name: MkDocs
     runs-on: "ubuntu-latest"
-    env:
-      ACTIONS_ALLOW_UNSECURE_COMMANDS: true
     steps:
       - uses: actions/checkout@v2
-      - uses: goanpeca/setup-miniconda@v1
+      - uses: conda-incubator/setup-miniconda@v2.1.1
```

Comment:
It was from before the author, goanpeca, moved it to the incubator. Definitely too old.